Exit code is incorrect on failure

When I run `cookiecutter` without a config file, it prints an error message to indicate the failure. But the exit code is still zero which causes issues in automation. E.g invocation and output:
```bash
ashoknn@PF0ZBVNJ:~/temp$ ./node_modules/.bin/cookiecutter
! Cannot find module '/home/ashoknn/temp/cookiecutter.config.js'
  at Function.Module._resolveFilename (module.js:543:15)
  at Function.Module._load (module.js:470:25)
  at Module.require (module.js:593:17)
  at require (internal/module.js:11:18)
  at getConfig (/home/ashoknn/bbgithub/temp/node_modules/cookiecutter/src/lib/config.js:103:20)
  at Promise.resolve.then (/home/ashoknn/temp/node_modules/cookiecutter/src/index.js:15:24)
  at <anonymous>
  at process._tickCallback (internal/process/next_tick.js:118:7)
  at Function.Module.runMain (module.js:692:11)
  at startup (bootstrap_node.js:194:16)

ashoknn@PF0ZBVNJ:~/temp$ echo $?
0
```

--- a/src/index.js
+++ b/src/index.js
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
 const inquirer = require('inquirer');
+const process = require('process');
 const {renderFiles} = require('./lib/renderer');
 const {makeFieldQuestions, makeTemplateQuestion} = require('./lib/questions');
 const {getConfig, getTemplateConfig} = require('./lib/config');
@@ -36,4 +37,5 @@ Promise.resolve()
     })
     .catch(e => {
         log.error(e);
+        process.exit(1);
     });

